ravedude/src/board.rs

```diff
@@ -272,6 +273,31 @@ impl Board for TrinketPro {
     }
 }
 
+struct Trinket;
+
+impl Board for Trinket {
+    fn display_name(&self) -> &str {
+        "Trinket"
+    }
+
+    fn needs_reset(&self) -> Option<&str> {
+        Some("Reset the board by pressing the reset button once.")
+    }
+
+    fn avrdude_options(&self) -> avrdude::AvrdudeOptions {
+        avrdude::AvrdudeOptions {
+            programmer: "usbtiny",
+            partno: "attiny85",
+            baudrate: None,
+            do_chip_erase: false,
```

Comment:
When I tried this on my board, I ran into issues... Were you successful with this (maybe my hardware is broken)?

Reply:
It succeeded at least once on blink example. Let me flash blink example few times and search results. Will test on MacBook and Ubuntu